setuptools_scm/git.py

```diff
@@ -4,7 +4,7 @@
 import warnings
 
 FILES_COMMAND = 'git ls-files'
-DEFAULT_DESCRIBE = 'git describe --tags --long --match *.*'
+DEFAULT_DESCRIBE = 'git describe --tags --long --dirty --match *.*'
```

Comment:
thats ok, but "works for me" is not a metric end-users appreciate ^^

for examples see #134